packages/client/src/__testSetup__/node.ts

```diff
@@ -1,21 +1,26 @@
 import { v4 as uuid } from "uuid";
 import { createUserWithEmailAndPassword, signOut } from "@firebase/auth";
 
-import { Collection } from "@eisbuk/shared";
+import { Collection, OrganizationData } from "@eisbuk/shared";
 
 import { adminDb, auth } from "./firestoreSetup";
 
 import { __withEmulators__ } from "@/__testUtils__/envUtils";
 
 interface SetUpOrganization {
-  (doLogin?: boolean): Promise<{
+  (doLogin?: boolean, additionalSetup?: Partial<OrganizationData>): Promise<{
     organization: string;
     email: string;
     pass: string;
   }>;
 }
 
-export const setUpOrganization: SetUpOrganization = async (doLogin = true) => {
+const smtpPort = 5000;
```

Comment:
@silviot I've moved this here as we won't need smtp setup while testing client code as tests testing cloud functions (in charge of sending emails) are always ran in node environment.